merge-openrpc.js

```diff
@@ -48,7 +48,9 @@ const unneeded = [
   /eth_sign/,
   /debug_.*/,
   /engine_.*/,
-  /eth_createAccessList/
+  /eth_createAccessList/,
+  /eth_getBlockReceipts/,
+  /eth_maxPriorityFeePerGas/,
```

Comment:
```suggestion
  /eth_maxPriorityFeePerGas/,
  /eth_blobBaseFee/,
  /eth_subscribe/,
```

Can we also filter out `eth_blobBaseFee` and `eth_subscribe` (since the standard version of `eth_subscribe` contains subscription types that cause errors)?

Reply:
eth_subscribe we have in our api-specs, heres a separate PR for that: https://github.com/MetaMask/api-specs/pull/250/files